ganttproject/src/biz/ganttproject/impex/google/GoogleAuth.java

```diff
@@ -30,76 +31,54 @@
   private static final String APPLICATION_NAME =
           "Google Export for GanttProject";
 
-  // Directory to store user credentials for this application
-  private static final java.io.File DATA_STORE_DIR = new java.io.File(
-          "data/resources/credentials");
-
-  // Global instance of the {@link FileDataStoreFactory}
-  private static FileDataStoreFactory DATA_STORE_FACTORY;
-
   // Global instance of the JSON factory
-  private static final JsonFactory JSON_FACTORY =
+  private final JsonFactory JSON_FACTORY =
           JacksonFactory.getDefaultInstance();
 
   // Global instance of the HTTP transport
-  private static HttpTransport HTTP_TRANSPORT;
+  private HttpTransport HTTP_TRANSPORT = new HttpTransport() {
+    @Override
+    protected LowLevelHttpRequest buildRequest(String method, String url) throws IOException {
+      return null;
+    }
+  };
 
   private static final List<String> SCOPES =
           Arrays.asList(CalendarScopes.CALENDAR_READONLY);
 
-  static {
-    try {
-      HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
-      DATA_STORE_FACTORY = new FileDataStoreFactory(DATA_STORE_DIR);
-    } catch (Throwable t) {
-      t.printStackTrace();
-    }
-  }
-
-  public static Credential authorize() {
-    try {
+  public Credential authorize() throws Exception{
       // Build flow and trigger user authorization request.
-      GoogleAuthorizationCodeFlow flow =
-              new GoogleAuthorizationCodeFlow.Builder(
-                      HTTP_TRANSPORT, JSON_FACTORY, CLIENT_ID, CLIENT_SECRET, SCOPES)
-                      .setDataStoreFactory(DATA_STORE_FACTORY)
-                      .setAccessType("offline")
-                      .build();
-      Credential credential = new AuthorizationCodeInstalledApp(
-              flow, new LocalServerReceiver()).authorize("ganttuser");
-      System.out.println(
-              "Credentials saved to " + DATA_STORE_DIR.getAbsolutePath());
-      return credential;
-    } catch (IOException e) {
-      e.printStackTrace();
-      return null;
-    }
+    HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+    GoogleAuthorizationCodeFlow flow =
+        new GoogleAuthorizationCodeFlow.Builder(
+            HTTP_TRANSPORT, JSON_FACTORY, CLIENT_ID, CLIENT_SECRET, SCOPES)
+            .build();
+    Credential credential = new AuthorizationCodeInstalledApp(
+        flow, new LocalServerReceiver()).authorize("ganttuser");
+    return credential;
   }
 
-  public static com.google.api.services.calendar.Calendar
-  getCalendarService() throws IOException {
+  public Calendar
+  getCalendarService() throws Exception {
```

Comment:
what's the point of breaking the line? And why `Exception` instead of `IOException`?